mails/themes/modern/core/guest_to_customer.html.twig

```diff
@@ -251,7 +251,8 @@
                                     <tr>
                                       <td align="left" style="font-size:0px;padding:10px 25px;padding-top:0px;word-break:break-word;">
                                         <div style="font-family:Open sans, arial, sans-serif;font-size:14px;line-height:25px;text-align:left;color:#363A41;">
-                                          <span class="label">{{ 'Email address:'|trans({}, 'Emails.Body', locale)|raw }}</span> <span style="color:#25B9D7;font-weight:600">{email}</span>
+                                          <span class="label">{{ 'Email address:'|trans({}, 'Emails.Body', locale)|raw }}</span> <span style="color:#25B9D7;font-weight:600">{email}</span><br><br>
```

Comment:
@MatShir, isn't it supposed to be removed?

Reply:
I think we still want to display the email address, what was removed (in another PR) is the generated password, because it is a bad practice to send passwords by mail.

@MatShir do you confirm ?

Reply:
@colinegin @marionf If that's ok I ping you because @MatShir will be off for a few days

Reply:
After discussing with @LouiseBonnard and @MatShir , indeed you can leave the email address in the email, just as it is done in the PR. We will study more in depth later the need to leave email addresses in such emails, just like in the account confirmation email. Thanks @matthieu-rolland !